### PR TITLE
Added us2 to the regions list, wasted 9 hours of my life.

### PR DESCRIPTION
**Pull request by RuhmitSahu**
The title is pretty self-explanatory, I added us2 to the regions list for ethash, while also keeping it removed from etchash.

### CODE REVIEW

File: src/mining.js [hunk 1]
```diff
@@ -276,19 +280,32 @@ async function selectPool(minerData, algo) {
...
 				console.log(chalk.green(`Only one pool available with these settings, using ${poolList[0].name}`))
 			}
 			presence.configuring("Selecting pool region");
-			const regionList = [];
+			var regionList = [];
+			if(algo == "etchash" || algo == "kawpow" || algo == "beamv3" || algo == "EQUI144_5" || algo == "randomx"){
+				regionList = [];
+			} else {
+			//	console.log(chalk.green(`AAAA ${poolList[1].name}`))
+			console.log(chalk.bold.cyan(`Ignore "us2" on NiceHash lol -Plerion`))
```
Comment (vtheskeleton) [inline src/mining.js:302]: bruh.

**Review by vtheskeleton — CHANGES_REQUESTED** [review]
This PR has a variety of issues
bad practices, leaving in debugging code in comments, it actually just not working right with it being shown on nicehash and just generally screwing up formatting

**Comment by vtheskeleton** [comment]
I dont feel like this pr is ready to be added at all
Make an issue on it and I can implement it myself

**Comment by vtheskeleton** [comment]
We have decided that this pr is too big of a mess

**Comment by RuhmitSahu** [comment]
ouch

**Pull request closed without merging**